src/olympia/editors/templates/editors/base.html

```diff
@@ -57,12 +57,8 @@ <h1 class="site-title amo">
         <a href="{{ url('editors.unlisted_queue') }}" class="controller">
           {{ _('Unlisted Queues') }}</a>
```

Comment:
Could collapse the menu into a link, like the performance and announcement links.